autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -163,31 +208,12 @@ function updateCourseCardAsync(
   index: number, courseCard: CourseCardOptions, term: string,
 ): ThunkAction<void, RootState, undefined, UpdateCourseAction> {
   return (dispatch): void => {
-    const [subject, courseNum] = courseCard.course.split(' ');
-
-    fetch(`/api/sections?dept=${subject}&course_num=${courseNum}&term=${term}`)
-      .then(
-        (res) => res.json(),
-      )
-      .then(
-        parseSectionSelected,
-      )
-      .then(
-        sortSections,
-      )
-      .then(
-        (sections) => {
-          const hasHonors = sections.some((section) => section.section.honors);
-          const hasWeb = sections.some((section) => section.section.web);
-          dispatch(updateCourseCardSync(index, {
-            ...courseCard, sections, hasHonors, hasWeb,
-          }));
-        },
-      )
-      .catch((error) => {
-        // eslint-disable-next-line no-console
-        console.log(error);
-      });
+    createCourseCardFrom(courseCard, term).then((updatedCourseCard) => {
+      dispatch(updateCourseCardSync(index, updatedCourseCard));
+    }).catch((error) => {
+      // eslint-disable-next-line no-console
+      console.log(error);
```

Comment:
I'm fine leaving this for now, but tbh we probably _shouldn't_ be outputting to the console - at least not in production. We can always add replace this with actual logging that would send this error somewhere we could view though (i.e. for crash analytics) / use a logging library so when we're debugging we can have it print to the console, but hide it when in prod.

Reply:
If you look at the diff I just moved this into a function so this console log was there before, I agree but I didn't want to change what was already there. Should I just remove it?

Reply:
I'm thinking, yeah, we can remove it for now. I only put it in there as like a "just in case" sorta thing, but we can figure out a better crash-handling system later